modules/process/manager.go

```diff
@@ -88,50 +108,58 @@ func (pm *Manager) AddContext(parent context.Context, description string) (ctx c
 func (pm *Manager) AddContextTimeout(parent context.Context, timeout time.Duration, description string) (ctx context.Context, cancel context.CancelFunc, finshed FinishedFunc) {
 	ctx, cancel = context.WithTimeout(parent, timeout)
 
-	ctx, pid, finshed := pm.Add(ctx, description, cancel)
+	ctx, _, finshed = pm.Add(ctx, description, cancel, NormalProcessType, true)
```

Comment:
I think we should be very careful here.

If users pass 0 to `timeout`, I think we should panic ASAP to report such low-level mistake.

Some code might call `AddContextTimeout(..., dur * time.Second, ...)` without checking the value of `dur`. And some developers may misundestand `0` as no-timeout like other frameworks.